include/pokeagb/overworld/script.h

```diff
@@ -214,10 +214,11 @@ extern u16 var_800B;
 extern u16 var_800C;
 
 /**
- * Script temporary variable 0x800D
+ * Script temporary variable 0x800D,
+ * which is the lastresult seen in scripts
  * @address{BPRE,020370BE}
  */
-extern u16 var_800D;
+extern u16 var_800D_lastresult;
```

Comment:
You shouldn't make breaking changes like this - renaming symbols affects other people's projects and will cause them not to compile. If you think a name could be better then you should file an issue instead.